package.json

```diff
@@ -12,7 +15,8 @@
     "lab": "^13.1.0",
     "pre-commit": "^1.2.2",
     "prettier": "^1.8.2",
-    "sinon": "^4.1.3"
+    "sinon": "^4.1.3",
+    "winston": "^3.1.0"
```

Comment:
I'm pretty sure `winston` is a regular dependency here, not a dev dependency.

Reply:
Oh yea.. when I first added it, it was purely for the tests, but now its the formatter, i'll get it fixed.